netlify.toml

```diff
@@ -144,3 +144,9 @@ HUGO_ENABLEGITINFO = "true"
   status = 301
   force = false
 
+[[redirects]]
+  from = "/docs/reference/artifacts/*"
+  to = "/docs/reference/artifacts-legacy/:splat"
+  status = 301
+  force = false
+
```

Comment:
Based on the old site's navigation, looks like we need an additional redirect

/reference/artifacts-with-artifactsrewrite/ moved to /docs/reference/ref-artifacts/


```
[[redirects]]
  from = "/reference/artifacts/ *"
  to = "/docs/reference/artifacts-legacy/:splat"
  status = 301
  force = false

[[redirects]]
  from = "/reference/artifacts-with-artifactsrewrite/ *"
  to = "/docs/reference/ref-artifacts/:splat"
  status = 301
  force = false
```

Reply:
addressed with 2ea79add951530d61609d2505da8b5e661f66757